### ISSUE TITLE
[BE] Autoclose issues that haven't been updated in X months

### ISSUE THREAD

**Issue by ZainRizvi** [pytorch/pytorch#125626]
In order to avoid a giant list of issues that no one has touched in over a year and no one cares about anymore, let's have a bot auto-ping issues after X months warning of an impending closure, and then actually close the issue after another month of inactivity

**Comment by kit1980** [comment]
This is basically the same as https://github.com/pytorch/pytorch/issues/124016

**Comment by huydhn** [comment]
To clarify, this is only for CI-related issues in https://github.com/orgs/pytorch/projects/15, not PyTorch issues in general.  We want this to keep our project board sane and get rid of old and irrelevant issues.

**Comment by kit1980** [comment]
> To clarify, this is only for CI-related issues in https://github.com/orgs/pytorch/projects/15, not PyTorch issues in general. We want this to keep our project board sane and get rid of old and irrelevant issues.

Reading the issue again, it doesn't look like this only for CI-related issues.
Maybe rename/rephrase if this really was the intention.

**Comment by kit1980** [comment]
Also, if this really only about project/15, why spend time on a bot - we can just close them manually.

**Comment by huydhn** [comment]
> Also, if this really only about projects/15, why spend time on a bot - we can just close them manually.

Yup, that was one of the option @ZainRizvi and I went over today.  We will schedule a session to go over the exercise and see how it goes.  The process roughly looks like this:

1. Identify irrelevant CI issues older than X months.
2. Ping people about it.
3. If no reply is received within N days, run the script to close the issue.
